docker/Dockerfile

```diff
@@ -1,24 +1,27 @@
 ARG VERSION
 
-FROM ghcr.io/deephaven/server:${VERSION}
+FROM ghcr.io/deephaven/server-sklearn:${VERSION}
 
 ARG DH_IB_VERSION
 ARG DH_IB_BRANCH
 
+COPY deephaven_ib-0.2.4-py3-none-any.whl /tmp/
```

Comment:
The prior version of the dockerfile checked out the repo, built the whl, and installed the whl.  That functionality should be reestablished, instead of just copying in some static version.

If you need the ability to pull from a different repo, you can make the repo path an ARG.